graphql/schema.graphql

```diff
@@ -266,6 +265,7 @@ type Mutation {
     @withSession
     @withPermissions
     @cacheControl(scope: PRIVATE)
+    @checkUserAccess
```

Comment:
What does the `@withPermissions` directive (two lines above) check? Asking so we can avoid calling store-permissions/vtexid twice unnecessarily.

Reply:
It calls the `checkUserPermission` in the storefront-permission, so at the moment we are not reviewing the permission flow, just the authentication.

Reply:
Got it, the permissions check is the next level, then, doing authorization instead of authentication.

In that case it seems like `checkUserPermission` should always run before `withPermissions` when both exist. It doesn't make sense to check permissions for an unauthenticated or invalidly authenticated user.

Reply:
Yes. Are the directives executed in sequence?

Reply:
Yep!